niworkflows/interfaces/tests/test_bids.py

```diff
@@ -291,24 +295,41 @@ def test_DerivativesDataSink_build_path(
 
         ds_inputs.append(str(fname))
 
-    dds = bintfs.DerivativesDataSink(
+    base_directory = tmp_path / "output"
+    work_dir = tmp_path / "work"
+    base_directory.mkdir()
+    work_dir.mkdir()
```

Comment:
Unused, but I don't want to start CI over again. Apply before merging.

```suggestion
    base_directory.mkdir()
```